_people/vincent_spada.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Vincent Spada
+position: undergraduate
+avatar: vincent_spada.png
+---
+
+<img width="300" src="{{site.baseurl}}/images/people/{{page.avatar}}" data-action="zoom">
+
+- _Undergraduate, Aerospace Engineering, UT Austin_<br>
+
+<i class="fa fa-envelope-o"></i> `vinspada@utexas.edu`<br>
+<i class="fa fa-newspaper-o"></i> [Resume (pdf)](/documents/Spada.Vincent.Resume.pdf)<br>

```

Comment:
The pdf file name here does not match the one being uploaded to the `documents` folder.